src/proc_fuse.c

```diff
@@ -145,8 +145,11 @@ __lxcfs_fuse_ops int proc_getattr(const char *path, struct stat *sb)
 	    strcmp(path, "/proc/swaps")		== 0 ||
 	    strcmp(path, "/proc/loadavg")	== 0 ||
 	    strcmp(path, "/proc/slabinfo")	== 0) {
-		if (liblxcfs_functional())
+		if (liblxcfs_functional()) {
+			if (!is_ptrace_allowed())
+				return log_error(-EACCES, RETRICTED_YAMA_PTRACE_POLICY);
```

Comment:
Should probably be RESTRICTED rather than RETRICTED

Reply:
Though that likely doesn't matter too much as we'll want to use the much simpler approach that's not yama-specific anyways :)

Reply:
Fixed, for posterity :upside_down_face: